easybuild/easyconfigs/i/IQ-TREE/IQ-TREE-1.6.6-intel-2018a.eb

```diff
@@ -0,0 +1,33 @@
+easyblock = 'CMakeMake'
+
+name = 'IQ-TREE'
+version = '1.6.6'
+
+homepage = 'http://www.iqtree.org/'
+description = """Efficient phylogenomic software by maximum likelihood"""
+
+toolchain = {'name': 'intel', 'version': '2018a'}
+toolchainopts = {'usempi': True}
+
+github_account = 'Cibiv'
+source_urls = [GITHUB_SOURCE]
+sources = ['v%(version)s.tar.gz']
+checksums = ['31bc44e3a79edb889f3e2ed396d1492b8e38d93437b2ae64deb452867929dc56']
+
+builddependencies = [
+    ('CMake', '3.10.2'),
+    ('Eigen', '3.3.4', '', True),
```

Comment:
Is there a particular reason we need to use `Eigen` built with `dummy`?

Reply:
@vanzod `Eigen` is only header files, there's absolutely no point in using anything else than `dummy`